### ISSUE TITLE
The documentation is very hard to follow

### ISSUE THREAD

**Issue by KennethLavrsen** [custom-components/alexa_media_player#153]
Hi just installed this custom component and I had huge issues with the installation and I still do not understand how to use all of it

First the installation. The copying of the files and the setting in the config is clear.
But the installation fails because I lack alexapy. It took me a long time to figure this out. My guess is that everyone installing this the first time will have this error. It should be part of the installation steps.

The next is once you restart HA nothing happens. And it was pure guess work and one hour of grief that I figured out that I had to go to the entities and find a configuration item which showed an image that could not be loaded and something that looked like a capcha. Well I took the URL for the image and figured out it was an expired image so I restarted HA hurried to the config entity and got it to work.  It was pure luck that I found it. This needs to be described in the installation steps.

And then we get to configurations. The examples are mostly the tts which is deprecated. All the examples should be updated to the future proof way.

And on the Notification component the syntax is JSON. Where am I supposed to put that? Why is it not YAML? This page I do not understand at all.

I have been fighting with the YAML examples for scripting the TTS using notify.alexa_media. The thing that catch you is that there is a data below the data and figuring out where to change data to data_template took me a long time. And I am still not sure why it works for me now. More examples would be good and examples using the future proof way and not the deprecated way

**Comment by alandtse** [comment]
@KennethLavrsen, thanks for the comments.  The wiki documentation is open for anyone to edit so feel free to make edits as necessary to clarify the documentation.  This is an open source effort so all contributions are welcome.  The documentation is currently written in a way that developers understand it, but obviously it may miss key points.

To answer some of your specific points:

The alexapy dependency issue only fails for certain use cases (primarily hass.io) but it's not consistent. I use venv and I've never had it fail. Others with hass.io report no issues.  That said, it may be worth pointing out in the docs.

The Captcha should trigger a HA notification. That can probably be updated in the wiki to be more explicit since HA has made notifications more hidden (before they were unmissable).

Notification in JSON because that's the dev-service page. Feel free to convert to YAML.  The difference between data and data_template is entirely based on how HA want to implement it.  Again, as a developer, I think we assume people understand HA's YAML conventions.

**Comment by almostserious** [comment]
I just added a small disclaimer in the installation instruction for hassio users. As i was also just struggeling with it, and then remembered that i need to restart hassio two times for the alexapy dependency to load before the component gets loaded.

**Comment by alandtse** [comment]
Thanks @almostserious.

**Comment by G8YTZ** [comment]
I too found the documentation hard to follow, especially as I am running HASS.IO in Docker on Ubuntu and the instructions don't really consider this configuration.

I use HASS.IO in YAML Mode, so I have SAMBA Share installed, so I have the CONFIG directory as an NFS mount on my MacBook. 

**To install this fantastic piece of code** you just need to download the directory of files as a ZIP and copy the structure across to the config/custom_components directory, when updating, just overnight the old files, but I also rename the old directory as old_alexa_media until everything is working again.

<img width="1089" alt="Screenshot 2019-08-10 at 12 15 59" src="https://user-images.githubusercontent.com/35895759/62821178-ad4cdf80-bb68-11e9-964f-687871eeb409.png">

**Comment by jamman9000** [comment]
Seems this thread might be appropriate for my post:

@aptonline From another thread you have it seems like you have the running of routines figured out, could you take a look at the instruction page and edit it if you see issues with it? Not having much luck myself, dev tools service page won't let me throw more keys at the service than what's allowed.
